cg/cli/upload/scout.py

```diff
@@ -173,16 +188,20 @@ def validate_case_samples_are_rna(context: CGConfig, case_id: str) -> None:
 @click.option("-r", "--research", is_flag=True, help="Upload research report instead of clinical")
 @click.argument("case_id")
 @click.pass_context
-def upload_rna_to_scout(context, case_id: str, dry_run: bool, research: bool) -> None:
-    """Upload an RNA case's gene fusion report and junction splice files for all samples connect via subject ID."""
+def upload_rna_to_scout(
+    context, case_id: str, dry_run: bool, research: bool, analysis: str, re_upload: bool
+) -> None:
+    """Upload an RNA case's gene fusion report and junction splice files for all samples connected via subject ID."""
     LOG.info("----------------- UPLOAD RNA TO SCOUT -----------------------")
 
     context.invoke(validate_case_samples_are_rna, case_id=case_id)
     context.invoke(upload_rna_alignment_file_to_scout, case_id=case_id, dry_run=dry_run)
     context.invoke(upload_multiqc_to_scout, case_id=case_id, dry_run=dry_run)
-    context.invoke(
-        upload_rna_fusion_report_to_scout, case_id=case_id, dry_run=dry_run, research=research
-    )
+    if analysis != Workflow.TOMTE:
+        context.invoke(
+            upload_rna_fusion_report_to_scout, case_id=case_id, dry_run=dry_run, research=research
+        )
+    context.invoke(upload_omics_to_scout, case_id=case_id, dry_run=dry_run)
```

Comment:
Similarly here, I would prefer not to have this command do different things for different workflows. Since this is a CLI command we cannot use the same inheritance pattern to split the functionality. 

Instead I would change this function to be `upload_mip_rna_to_scout` and create a new function `upload_tomte_to_scout` which would implement the exact logic you need.

Reply:
created a tomte-to-scout command, no need for the functions as all exists in cli commands